src/blueprint/into_schema.rs

```diff
@@ -30,14 +35,46 @@ fn to_type_ref(type_of: &Type) -> dynamic::TypeRef {
   }
 }
 
+fn get_cache_key<'a, H: Hasher>(ctx: &'a EvaluationContext<'a, ResolverContext<'a>>, mut hasher: H) -> u64 {
+  let state = &mut hasher;
+  // Hash on parent value
+  if let Some(const_value) = ctx
+    .graphql_ctx
+    .parent_value
+    .as_value()
+    // TODO: handle _id, id, or any field that has @key on it.
+    .and_then(|data| data.get_key("id"))
```

Comment:
- [ ] Should keep going up till you keep finding a parent.
- [ ] What happens if `id` has not been queried?

Reply:
Not handling the first case as of now. For the second one, if parent's `id` isn't queried then not caching the value